docs/src/pages/components/text-fields/LayoutTextFields.js

```diff
@@ -1,29 +1,22 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import TextField from '@material-ui/core/TextField';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    display: 'flex',
-    flexWrap: 'wrap',
-  },
-  textField: {
-    marginLeft: theme.spacing(1),
-    marginRight: theme.spacing(1),
-    width: '25ch',
-  },
-}));
-
 export default function LayoutTextFields() {
-  const classes = useStyles();
-
   return (
-    <div className={classes.root}>
+    <Box
+      sx={{
+        display: 'flex',
+        flexWrap: 'wrap',
+        '& > :not(style)': { mb: 2 },
+        '& .MuiTextField-root': { m: 1 },
+        '& .MuiTextField-root:not(:first-of-type)': { width: '25ch' },
```

Comment:
👍 nice cleanup

Reply:
I tried to do it in other examples, but I end up giving more problems than solutions